src/resolvers.ts

```diff
@@ -1,72 +1,56 @@
-import { addFacility, getFacilityById, searchFacilities } from './services/facilityService'
-import { addHealthcareProfessional, 
-    getHealthcareProfessionalById,
-    searchHealthcareProfessionals } from './services/healthcareProfessionalService'
-import {
-    Contact,
-    Degree,
-    Facility,
-    Insurance,
-    HealthcareProfessional,
-    Specialty,
-    LocaleNameInput,
-    HealthcareProfessionalInput,
-    SpokenLanguage,
-    LocaleName,
-    DegreeInput,
-    SpecialtyInput,
-    SpokenLanguageInput,
-    FacilityInput,
-    ContactInput
-} from './typeDefs/gqlTypes'
+import * as facility from './services/facilityService'
```

Comment:
Should make it clear that these are from a service. facility just sounds like an object/entity.

```suggestion
import * as facilityService from './services/facilityService'
```